scripts/learn_kind_images.sh

```diff
@@ -98,6 +98,7 @@ echo "" >> "${kind_versionfile}.tmp"
 
 release_json=$(curl -fsSL "https://api.github.com/repos/kubernetes-sigs/kind/releases/tags/${kind_version}"| jq '
   [ .body  | capture("- v?1\\.(?<minor>[0-9]+)(.(?<patch>[0-9]+))?: `kindest/node:v(?<version>[^@]+)@sha256:(?<sha256>[^`]+)`\r"; "g") ]
+  | group_by(.minor) | map(max_by(.patch))
```

Comment:
kind lists multiple patch versions for some of the minor releases.
We just select the latest (max) patch version for each.